core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -282,7 +282,8 @@ object Boot extends IOApp with LazyLogging {
           requesterPaysRole,
           dmConfig,
           projectTemplate,
-          servicePerimeterService
+          servicePerimeterService,
+          RawlsBillingAccountName(gcsConfig.getString("adminRegisterBillingAccountId"))
```

Comment:
We looked at abstracting out a GCS config, but decided that would blow up very quickly on us.